src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Inform.cshtml

```diff
@@ -18,9 +18,19 @@
                 <li>Review and approve applications</li>
                 <li>Connect with your chosen business and confirm details of the transfer</li>
             </ol>
-            <p>
-                <br />
-            </p>
```

Comment:
One tiny thing, the spacing provided by this break separated the paragraph from the buttons at the bottom. This needs to be re-introduced below your new paragraph.

Reply:
added back in but used a non breaking space as it's more semantically meaningful I think than a BR